src/store/getters.ts

```diff
@@ -72,6 +74,8 @@ export default {
       useCache?: boolean;
       accountType?: AccountType;
       accountIndex?: number;
+      chainCode?: string | undefined;
```

Comment:
`chainCode?: string | undefined`
is the same as `chainCode?: string` ? 
`?` includes undefined

Also make sure this parameter matches the same format as in `client.ts`

Reply:
fixed